easysocial/easysocial/groupinvite.php

```diff
@@ -105,23 +95,39 @@ public function delete()
 			switch ($operation)
 			{
 				case 'leave':
+								// Remove the user from the group/page.
+								$res->result->status = $cluster->leave($user->id);
+
+								if (!$res->result->status)
+								{
+									$res->result->status	=	0;
+									$res->result->message	=	JText::_('PLG_API_EASYSOCIAL_INVALID_USER_MESSAGE');
+								}
+								else
+								{
+									// Notify group/page members
+									$cluster->notifyMembers('leave', array('userId' => $my->id));
+									$res->result->message	=	JText::_('PLG_API_EASYSOCIAL_LEAVE_GROUP_MESSAGE');
+									break;
+								}
 
-	// Remove the user from the group.
-								$group->leave($user->id);
-
-	// Notify group members
-								$group->notifyMembers('leave', array('userId' => $my->id));
-								$res->result->message	=	JText::_('PLG_API_EASYSOCIAL_LEAVE_GROUP_MESSAGE');
-								break;
 				case 'remove':
 
-	// Remove the user from the group.
-								$group->deleteMember($user->id);
-
-								// Notify group member
-								$group->notifyMembers('user.remove', array('userId' => $user->id));
-								$res->result->message	=	JText::_('PLG_API_EASYSOCIAL_USER_REMOVE_SUCCESS_MESSAGE');
-								break;
+								// Remove the user from the group/page.
+								$res->result->status = $cluster->deleteMember($user->id);
+
+								if (!$res->result->status)
+								{
+									$res->result->status	=	0;
+									$res->result->message	=	JText::_('PLG_API_EASYSOCIAL_INVALID_USER_MESSAGE');
+								}
+								else
+								{
+									// Notify group/page member
+									$cluster->notifyMembers('user.remove', array('userId' => $user->id));
+									$res->result->message	=	JText::_('PLG_API_EASYSOCIAL_USER_REMOVE_SUCCESS_MESSAGE');
+									break;
```

Comment:
@Pratikshakhandagale  - Check if you really need `break` statement inside `else` block